README.md

```diff
@@ -94,6 +94,9 @@ It stores spans as json and has been designed for larger scale.
 
 Note: This store requires a [spark job](https://github.com/openzipkin/zipkin-dependencies) to aggregate dependency links.
 
+### Throttling
```

Comment:
I can see why you accidentally put this section in the top-level readme as opposed to the server readme. The "disabling endpoints" readme was accidentally put in the wrong readme

Reply:
There is also a comment in the zipkin-server README.  Would you like me to simply remove this one and not touch the other?

Reply:
sgtm